doc/howtos/OSTree.rst

```diff
@@ -0,0 +1,153 @@
+do this by incuding the following configuration snippet in your local.conf
+or other suitable global Yocto configuration file::
+
+    REFKIT_IMAGE_EXTRA_FEATURES += "ostree"
+
+You can generate a pair of signing keys with the following command::
+
+    meta-refkit-core/files/gnupg/generate-keys.sh $(pwd)/gpg release@example.org
+
+This will generate a pair of private and public GPG keys with the key ID
+release@example.org, put them into the GPG keyring at $(pwd)/gpg and also
+export them to the files $(pwd)/release@example.org.{pub,sec}.
+
+To use these newly generated keys for signing and verifying the OSTree
+repository commits, add the following to your local.conf or other suitable
+global Yocto configuration file (replace $(pwd) with the path to your
+top-level intel-iot-refkit directory)::
+
+    OSTREE_GPGDIR = "$(pwd)/gpg"
+    OSTREE_GPGID  = "release@example.org"
+
+Assuming you want to use your build machine, build.example.org, in this
+example also as your update server, you can readily point the image for
+updates to your update/build server by adding the following to your
+local.conf or other suitable global Yocto configuration file::
+
+    OSTREE_REMOTE = "http://build.example.org/ostree/"
+
+You can also use HTTPS instead of HTTP if you want to and your server is
+properly configured for serving HTTPS requests.
+
+Next you need to expose the OSTree repository your builds are exported to
+over HTTP/HTTPS for clients to consume. By default this repository is
+located in build/tmp-glibc/deploy/ostree-repo, but you can change this
+location by adding the following to your local.conf or other suitable
+global Yocto configuration file::
+
+    OSTREE_EXPORT = "<path-to-the-repository>"
+
+Now assuming, you did not change the location, and you use Apache for
+service HTTP/HTTPS requests, you can expose this repository with Apache
+by adding the following to your Apache configuration::
+
+    Alias "/ostree/" "<path-to-intel-iot-refkit>/build/tmp-glibc/deploy/ostree-repo/"
+    
+    <Directory <path-to-inte-iot-refkit>/build/tmp-glibc/deploy/ostree-repo>
+        Options Indexes FollowSymLinks
+        Require all granted
+    </Directory>
+
+Finally you should restart (or start) your Apache server to activate the
+configuration changes. This might be a good time to also make sure that
+any firewall rules you might have will allow your clients access to the
+HTTP port of the server.
+
+Another alternative is to use the built-in trivial HTTP server in ostree
+available as the *ostree trivial-httpd* command (if it is enabled at
+compile time). With that you could serve out the repository with the
+following commands::
+
+    cd build/tmp-glibc/deploy
+    ln -sf ostree-repo ostree
+    ostree trivial-httpd --port 80
+
+A third alternative is use a simple Python HTTP server, for instance the
+one from the Yocto project at::
+
+    http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/lib/oeqa/utils/httpserver.py
```

Comment:
Actually, "httpserver.py", is part of OpenEmbedded core more than it can be said to be part of the Yocto Project.

Better just say "OpenEmbedded core" and link to http://git.openembedded.org/openembedded-core/tree/meta/lib/oeqa/utils/httpserver.py